app/scripts/modules/core/src/projects/configure/ConfigureProjectModal.tsx

```diff
@@ -24,6 +24,7 @@ import './ConfigureProjectModal.css';
 export interface IConfigureProjectModalProps extends IModalComponentProps {
   title: string;
   projectConfiguration: IProject;
+  isTeams?: boolean;
```

Comment:
Maybe instead of an `isTeams` boolean we can have a discriminator like `type?: 'team' | 'project'`

Reply:
+1. will fix.